common-rendering/src/components/keyEvents.stories.tsx

```diff
@@ -68,26 +59,43 @@ const events: KeyEvent[] = [
 	},
 ];
 
-const Default = (args: { theme: Theme }) => (
-	<KeyEvents keyEvents={events} {...args} />
+const keyEventComp = (dark: boolean, theme: Theme, title: string) => (
+	<div css={flexItemStyles}>
+		<div>{title}</div>
+		<KeyEvents keyEvents={events} theme={theme} supportsDarkMode={dark} />
+	</div>
 );
 
-Default.args = {
-	theme: Pillar.News,
-};
+const flexItemStyles = css`
+	flex-grow: 1;
+`;
+
+const keyEventWithTheme = (dark: boolean) => () => (
+	<div
+		css={css`
+			display: flex;
+			flex-direction: row;
+			justify-content: space-between;
+			flex-wrap: wrap;
+			-ms-flex-positive: 1;
```

Comment:
Is this needed? I think these browser specific properties get added in the build

Reply:
+1 I think line 80 can be removed :)